### PR TITLE
corrected a few style issues and fixed a few file names in tensorflow…

### PR DESCRIPTION
**Pull request by dmbala**
day-4/part-1-exercise-1 = minor style edit and sub title
day-4/part1-exercise-2 = minor style edit
day-4/part-1-exercise-3 = minor style edit
day-4/part1-exercise-4 = file names are changed in the tutorial example, redefined the tensorflow image path from the OSG, and minor style edit

### CODE REVIEW

File: docs/materials/day4/part1-ex4-singularity.md [hunk 1]
```diff
@@ -51,22 +52,25 @@ Requirements = HAS_SINGULARITY == True
...
 
 ``` console
 username@training $ condor_submit tf_matmul.submit 
 ```
 
-The job will look for a machine on the OSG that has singularity installed, creates the singularity container with the image `/cvmfs/singularity.opensciencegrid.org/tensorflow/tensorflow:latest` and executes the program `tf_matmul.py`.
+The job will look for a machine on the OSG that has singularity installed, On a matched machine, the creates the singularity container with the image `/cvmfs/singularity.opensciencegrid.org/opensciencegrid/tensorflow:latest` and executes the program `tf_matmul.py`.
 
-After the job is completed, you will see the output file `tf_matmul.output`.
+Once the finished, you will see the output file `tf_matmul.output`.
```
Comment (ChristinaLK) [inline docs/materials/day4/part1-ex4-singularity.md]: "Once the job"

**Comment by lmichael107** [comment]
looks like Bala needs to ignore (not stage changes for) his ci contents?

File: docs/materials/day4/part1-ex4-singularity.md [hunk 2]
```diff
@@ -51,22 +52,25 @@ Requirements = HAS_SINGULARITY == True
...
 
-Submit the job.
+Now submit the job to the OSG.
 
 ``` console
 username@training $ condor_submit tf_matmul.submit 
 ```
 
-The job will look for a machine on the OSG that has singularity installed, creates the singularity container with the image `/cvmfs/singularity.opensciencegrid.org/tensorflow/tensorflow:latest` and executes the program `tf_matmul.py`.
+The job will look for a machine on the OSG that has singularity installed, On a matched machine, the creates the singularity container with the image `/cvmfs/singularity.opensciencegrid.org/opensciencegrid/tensorflow:latest` and executes the program `tf_matmul.py`.
```
Comment (brianhlin) [inline docs/materials/day4/part1-ex4-singularity.md]: I'm a little confused by this sentence

Comment (dmbala) [reply docs/materials/day4/part1-ex4-singularity.md]: oops.. let me add the correction.

**Comment by brianhlin** [comment]
LGTM!

**Pull request merged**